src/teleop/gui/src/components/Carousel.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+<div>
+    <h3>Carousel Data</h3>
+    <div class="box1">
+        <Checkbox ref="open-loop" v-bind:name="'Open Loop'" v-on:toggle="openLoop = !openLoop" />
+        <div class="controls">
+            <div v-if="!openLoop">
+                <label for="position">Rotate carousel to position: </label>
+                <select v-model="site">
+                    <option>A</option>
+                    <option>B</option>
+                    <option>C</option>
+                </select>
+            </div>
+            <div v-else>
+                <button v-on:pointerdown="velocity = -1*velocityScaleDecimal" v-on:pointerup="velocity = 0" v-on:mouseout="velocity = 0">Reverse</button>
+                <button v-on:pointerdown="velocity = velocityScaleDecimal" v-on:pointerup="velocity = 0" v-on:mouseout="velocity = 0">Forward</button>
```

Comment:
I assume this is fine if the user clicks on the button, drags off (or onto the other button), and then releases? Just want to make sure someone has thought of that

Reply:
Would be nice to make the open loop control a component since it seems identical to the stuff in cache